### PR TITLE
Code spacing is a useful organizational aid.

### PR DESCRIPTION
**Pull request by niemeyer**

### CODE REVIEW

**Comment by johnsca** [comment]
Now that we have these style decisions made, it would be good to add linting to the Travis checks.

File: juju/framework.py [hunk 1]
```diff
@@ -580,7 +579,8 @@ def __get__(self, parent, parent_type=None):
             for attr_name, attr_value in parent_type.__dict__.items():
                 if attr_value is self:
                     if self.attr_name and attr_name != self.attr_name:
-                        raise RuntimeError("StoredState shared by {}.{} and {}.{}".format(parent_type.__name__, self.attr_name, parent_type.__name__, attr_name))
+                        parent_tname = parent_type.__name__
+                        raise RuntimeError("StoredState shared by {}.{} and {}.{}".format(parent_tname, self.attr_name, parent_tname, attr_name))
```
Comment (johnsca) [inline juju/framework.py]: Suggestion:

```python
parent_tname = parent_type.__name__
raise RuntimeError(f"StoredState shared by {parent_tname}.{self.attr_name} and {parent_tname}.{attr_name}")
```

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,11 @@
 language: python
 python:
-  - "3.6"
-  - "3.7"
+    - "3.6"
+    - "3.7"
+
+install:
+    - apt install flake8
```
Comment (johnsca) [inline .travis.yml]: `apt install python3-flake8`

File: juju/framework.py (hunk 1 above)
Comment (niemeyer) [reply juju/framework.py]: Done.

**Pull request merged**